main.go

```diff
@@ -94,9 +94,9 @@ func main() {
 	ioController.Logger().SetLevel(io.WarnLevel)
 	oktetoLog.Init(logrus.WarnLevel) // TODO: Remove when we fully move to ioController
 	if registrytoken.IsRegistryCredentialHelperCommand(os.Args) {
-		oktetoLog.SetOutput(os.Stderr)                  // TODO: Remove when we fully move to ioController
-		oktetoLog.SetLevel(oktetoLog.InfoLevel)         // TODO: Remove when we fully move to ioController
-		oktetoLog.SetOutputFormat(oktetoLog.JSONFormat) // TODO: Remove when we fully move to ioController
+		oktetoLog.SetOutput(os.Stderr)          // TODO: Remove when we fully move to ioController
+		oktetoLog.SetLevel(oktetoLog.InfoLevel) // TODO: Remove when we fully move to ioController
+		oktetoLog.SetOutputFormat("silent")
```

Comment:
I've got to not-show the logs when using this format, as in the `kubetoken` cmd

but

the stage log still shows as this is using the JSONFormat from the ioController, i believe we could have a "silent" format implemented for the ioController ?? @jLopezbarb was there a reason for not implementing this on the new controller? just to understand and perhaps create it to solve this.

@ifbyol @maroshii just for my understanding, is there a reason we need to have them json or could they be silent as in kubetoken?

Reply:
There was no reason. We didn't need it back at the time so we choose to only implement the minimal things. I'd add it if it's necessary to fix the issue without hesitation